src/test/java/org/folio/rest/impl/AccountsAPITest.java

```diff
@@ -149,20 +156,26 @@ public void eventIsPublishedWhenFeeFineIsClosedWithLoanAndNoRemainingAmount() {
     final JsonObject account = createAccountJsonObject(accountId)
       .put("loanId", loanId)
       .put("remaining", 90.00)
-      .put("status", createNamedObject("Open"));
+      .put("status", createNamedObject("Open"))
+      .put("dateClosed", null);
```

Comment:
Could you please explain, why do we need to put a null here? I've run the test and added account get request by id after creation of this account, there was no dateClosed field

Reply:
If I update the buildAccount function and add withDateClose(null), which is called by the createAccountJsonObject function, the JSON never returns the dateClosed field since it is initially null. So just for testing I add it here to verify that the property exists and then assess that the correct value has been added